app/assets/javascripts/accept_cookies_banner.js

```diff
@@ -0,0 +1,23 @@
+var chf = chf || {}
+$( document ).ready(function() {
+    chf.set_up_accept_cookies_banner();
+});
+chf.set_up_accept_cookies_banner = function () {
+    if (! chf.cookies_already_accepted_by_user()) {
+        jQuery('.accept_cookies_banner_nav').fadeIn(1000);
+    }
+    jQuery ('.i-accept-link').click(chf.user_accepts_our_cookies);
+}
+chf.cookies_already_accepted_by_user = function () {
+    return document.cookie.match(/user_accepts_our_cookies=true/) != null;
+}
+chf.user_accepts_our_cookies = function() {
+    document.cookie = "user_accepts_our_cookies=true; path=/"
```

Comment:
Seems overkill to include the JQuery cookie library just for this one line. But I'm open to doing it.